02-css/01-basic-layout/style.css

```diff
@@ -22,23 +22,32 @@ LAYOUT
 ===================*/
 
 #wrapper {
-
+  width: 960px;
+  margin: 0px auto;
 }
 
 header {
-
+  width: 100%;
+  text-align: center;
 }
 
 main {
-
+  width: 100%;
+  padding: 20px;
+  position: absolute;
```

Comment:
You can replace `position: absolute` with `float: left;` and main will contain its contents. Generally should avoid `absolute` unless you're actually positioning something.